app/src/test/java/org/simple/clinic/searchresultsview/PatientSearchViewControllerTest.kt

```diff
@@ -208,7 +210,129 @@ class PatientSearchViewControllerTest {
   private fun `params for search by parameters on register click`(): List<SearchPatientBy> {
     return listOf(
         SearchPatientBy.Name(searchText = patientName),
-        SearchPatientBy.PhoneNumber(searchText = "123456")
+        SearchPatientBy.PhoneNumber(searchText = phoneNumber)
     )
   }
+
+  @Test
+  fun `when searching patients by phone returns results, the results should be displayed`() {
```

Comment:
I plan to unify the tests for phone number and name search in a future PR.

Reply:
Will that be a parameterized test?

Reply:
I'm planning to make it one. I'll see what the final shape of the solution looks like when I get around to it.

Reply:
@ragunathjawahar This unification was done in #735